build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 }
 
 project.ext {
-    version_number = '2.2.0'
+    version_number = '2.3.0'
```

Comment:
I think the release scripts set this for you.

Reply:
Now addressed, thanks